Classes/GlobalStateExplorers/DatabaseBrowser/FLEXDatabaseManager.m

```diff
@@ -0,0 +1,194 @@
+//
+//  PTDatabaseManager.m
+//  PTDatabaseReader
+//
+//  Created by Peng Tao on 15/11/23.
+//  Copyright © 2015年 Peng Tao. All rights reserved.
+//
+
+#import "FLEXDatabaseManager.h"
+#import <sqlite3.h>
+
+
+static NSString *const QUERY_TABLENAMES_SQL = @"SELECT name FROM sqlite_master WHERE type='table' ORDER BY name";
+
+
+@implementation FLEXDatabaseManager
+{
+  sqlite3* _db;
+  NSString* _databasePath;
+}
+
+
+- (instancetype)initWithPath:(NSString*)aPath
+{  
+  self = [super init];
+  
+  if (self) {
+    _databasePath = [aPath copy];
+    _db           = nil;
+  }
+  return self;
+}
+
+
+- (BOOL)open {
```

Comment:
Is this pulled from FMDB or somewhere else? That's probably ok given the licensing from FMDB should permit this use, but we should definitely add a note in the Thanks & Credits section of the README, and maybe a comment around code that was copied with a link back to the original source.